yarn-project/archiver/src/archiver/archiver_store_test_suite.ts

```diff
@@ -115,7 +115,8 @@ export function describeArchiverDataStore(testName: string, getStore: () => Arch
         } satisfies ArchiverL1SynchPoint);
       });
 
-      it('returns the L1 block number that most recently added messages from inbox', async () => {
+      it.skip('returns the L1 block number that most recently added messages from inbox', async () => {
```

Comment:
@benesjan @LHerskind you guys understand what this test actually does? It seems to create a messageSent event at l2block 0 (which is lesser than INITIAL_L2_BLOCK_NUM)

Reply:
Some mock test checking that the store works, but here using gibberish as you point out.